### DISCUSSION TITLE
Re-use pan_id and channel after Z2M migration from docker to addon

### DISCUSSION THREAD

**Discussion by silver0007** [Koenkk/zigbee2mqtt#5513] [Devices]
I moved zigbee2mqtt from docker into HA addon. Unfortunately when I try to start z2m with the same pan_id and channel as before I get an error:

_Coordinator failed to start, probably the panID is already in use, try a different panID or channel_

Changing either pan_id or channel fix the problem, but requires repairing devices. Is it possible to keep the same settings and avoid re-pairing?
I tried to re-flash my CC1352 but it didn't help.

**Comment by Koenkk** [comment]
How to migrate is explained here: https://www.zigbee2mqtt.io/information/FAQ.html#how-do-i-move-my-zigbee2mqtt-instance-to-a-different-environment

It's important not to reflash your coordinator. I see you already did this. In this case put back the old `coordinator_backup.json` in the data directory, reflash your coordinator again and start z2m. 

If this is not done correctly you will get the pan_id error. If you don't have the old coordinator_backup.json you need to increase the pan_id as explained here: https://www.zigbee2mqtt.io/information/FAQ.html#how-do-i-migrate-from-a-cc2531-to-a-more-powerful-coordinator-eg-zzh and repair all your devices.

**Comment by silver0007** [comment]
Thanks for support, but in the meantime I managed to solve the problem in a bit different way. I was wondering how CC1352 knows old pan_id and channel and found somewhere info that it gets it from devices that are running around. I turned off all the fuses in my house except of the one holding HA hardware and took away all devices powered by batteries. Then tried to start z2m with old database.db. It started without any errors about pan_id/channel being in use. All old devices were discovered as soon as I turned on the fuses. There was no need to repair.
Hope it will help someone in the future and we can consider the topic as closed.

**Reply by Koenkk** [reply]
>  I was wondering how CC1352 knows old pan_id and channel and found somewhere info that it gets it from devices that are running around.

It does a scan right before starting.
